src/main/java/org/kohsuke/github/GHVerification.java

```diff
@@ -0,0 +1,52 @@
+package org.kohsuke.github;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+
+/**
+ * The commit/tag would be signed by user. This object would hold the verification status. Whether the Commit/Tag is
+ * signed or not.
+ *
+ * @author Sourabh Sarvotham Parkala
+ */
+@SuppressFBWarnings(value = { "UWF_UNWRITTEN_PUBLIC_OR_PROTECTED_FIELD", "UWF_UNWRITTEN_FIELD", "NP_UNWRITTEN_FIELD" },
+        justification = "JSON API")
+public class GHVerification {
+    private String reason, signature, payload;
```

Comment:
Would it make sense to create a enum which holds all possible reasons? The following page defines possible reasons (please see the "Signature verification object" section):

https://developer.github.com/v3/repos/commits/